lambda-create.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+aws lambda create-function \
+    --function-name "aws_fizzbuzzx" \
+    --runtime "python3.8" \
+    --role "arn:aws:iam::019263404189:role/fizzbuzzx_role" \
+    --handler "aws_function.lambda_handler" \
+    --zip-file "fileb://aws_fizzbuzzx.zip" \
```

Comment:
where does the zip file come from? does the cli automagicly do that?

Reply:
It is a way to create a deployment package for aws. The cli does not do it for us automagicly :(

More info here: https://docs.aws.amazon.com/lambda/latest/dg/python-package.html

Reply:
um, can you add the command(s) to generate the zip file?